java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/util/BatchJobUtil.scala

```diff
@@ -0,0 +1,46 @@
+package com._4paradigm.openmldb.taskmanager.util
+
+import java.io.File
+
+object BatchJobUtil {
+
+  /**
+   * Get the default batch jor from presupposed directories.
+   *
+   * @return the path of openmldb-batchjob jar
+   */
+  def findLocalBatchJobJar(): String = {
+    var jarPath = findBatchJobJar("../lib/")
+    if (jarPath == null) {
+      jarPath = findBatchJobJar("./openmldb-batchjob/target/")
+      if (jarPath == null) {
+        throw new Exception("Fail to find default batch job jar")
+      }
+    }
+
+    jarPath
+  }
+
+  /**
+   * Find the openmldb-batchjob jar from specified directory.
+   *
+   * @param libDirectory the directory to check
+   * @return
+   */
+  def findBatchJobJar(libDirectory: String): String = {
+    val libDirectoryFile = new File(libDirectory)
```

Comment:
will there any Exceptions ? e.g `FileNotFoundException`, `IOException`

Reply:
`IOException` could be better. But we may unified all the exceptions later.